### DISCUSSION TITLE
Hasura CLI Install Issues

### DISCUSSION THREAD

**Discussion by udigcowan** [hasura/graphql-engine#10162] [General]
Recently got a new work laptop with Windows 11 and WSL on it. Tried to do the windows install of the hasura cli from the instructions found here: https://hasura.io/docs/latest/hasura-cli/install-hasura-cli/. Not successful, when I run the command hasura, I get an error saying hasura cannot be found. Has anyone else run into this?

**Comment by meetzaveri** [comment]
Hello @udigcowan 

You will need to add Hasura CLI path to system PATH environment variable in Windows. This would enable you to use CLI directly from any sub-folder path when executing via terminal.

StackOverflow reference - [Adding a directory to the PATH environment variable in Windows](https://stackoverflow.com/questions/9546324/adding-a-directory-to-the-path-environment-variable-in-windows)

This is also mentioned in docs here under "windows" tab - https://hasura.io/docs/latest/hasura-cli/install-hasura-cli/#install-the-hasura-cli-binary-globally (see screenshot for reference)

<img width="829" alt="CLI_instructions_windows" src="https://github.com/hasura/graphql-engine/assets/21293048/9949ff75-7d1d-4e56-867f-1ebae42d3f72">
